system/RESTful/ResourceController.php

```diff
@@ -23,7 +24,7 @@ class ResourceController extends BaseResource
     /**
      * Return an array of resource objects, themselves in array format
      *
-     * @return mixed
+     * @return Response
```

Comment:
All of the RESTful methods are placeholders to be extended (optionally) so even though their current return value is `Response` they could also return `string` or possibly even nothing. If we type these here without including those then developers will need to retype the docblocks or risk breaking static analysis. 

My suggestion is to make all these `Response|string|null`.

Reply:
Changed to `Response|string|void`.